docs/rpc/grpc.md

```diff
@@ -1,81 +0,0 @@
-<!--- Hugo front matter used to generate the website version of this page:
-linkTitle: gRPC
---->
-
-# Semantic Conventions for gRPC
```

Comment:
This document shouldn't be removed, as it's also covers gRPC spans (which aren't covered by the proposal in the gRPC repository).

Reply:
reverted